### PR TITLE
Add support for specifying random seed

### PR DESCRIPTION
**Pull request by tyszw**
Enabled users to specify random seed value. 
Please see 048c0690644f63a7cf227d6943816cb5b4a235a3 for detail.

### CODE REVIEW

File: run.py [hunk 1]
```diff
@@ -156,6 +157,9 @@ def main():
     if args.debug:
         conf['fix_random_seed'] = True
 
+    if conf['random_seed'] != -1:
+        conf['fix_random_seed'] = True
+
```
Comment (sishida21) [inline run.py:162]:
```
    if args.debug:
        conf['fix_random_seed'] = True
        conf['random_seed'] = 1234

    if conf['random_seed'] != -1:
        conf['fix_random_seed'] = True
```

File: run.py [hunk 2]
```diff
@@ -190,7 +194,11 @@ def main():
             
     conf['filter_list'] = get_filter_modules(conf)
 
-    conf['random_generator'] = default_rng(1234) if conf['fix_random_seed'] else default_rng()
+    if conf['fix_random_seed']:
+        random_seed = conf['random_seed'] if conf['random_seed'] != -1 else 1234
+        conf['random_generator'] = default_rng(random_seed)
+    else:
+        conf['random_generator'] = default_rng()
```
Comment (sishida21) [inline run.py:201]:
```
conf['random_generator'] = default_rng(conf['random_seed']) if conf['fix_random_seed'] else default_rng()
```

**Comment by sishida21** [comment]
Please check the above modifications and revise the scripts.

**Comment by tyszw** [comment]
Thank you for your suggestions.
I have finished the revision, please check 3dcae755c878310e0baf16c2b2fbf1fb2fcfd028.

**Pull request merged**